inference-engine/scripts/build-local.sh

```diff
@@ -0,0 +1,50 @@
+#!/bin/bash
```

Comment:
Thought: The general rule of thumb is that if a script starts to get complicated, it should be written in python. These seem to be fairly simple, and are following the practices of the existing repo being ported over, so I think we're still fine to keep these as bash.

Reply:
I'd be happy to convert them to Python!

It sounds nicer anyway. I kept them as bash to match the pre-existing `build-wasm.sh`.

But after the merge, I wouldn't mind doing a bash-to-Python cleanup. 

I would rather have a Python ecosystem.

Reply:
This is a build script and it's pretty much an industry standard to write such scripts in bash. I suggest we leave them alone unless there is some really complex logic and it's getting hard to maintain them.